source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,78 @@
+.. _ruby-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: installation, setup, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you being developing, ensure you have the following dependencies
+      installed in your development environment:
+      
+      - `Ruby <https://www.ruby-lang.org/en/downloads/>`__ version 2.7
+        or later
+      - `bson <https://rubygems.org/gems/bson>`__ versions 4.14.1 to 6.0
```

Comment:
Q: do they need all of these `bson` versions?

you could consider making this a step in the following procedure, since that's also where they install the `mongo` gem

Reply:
No, I meant the version must be greater than 4.14.1 and less than 6.0